### PR TITLE
Setup coverage

### PR DESCRIPTION
**Pull request by gst**
should give us, (once the coveralls.io workers will process it), a coveralls.io badge with some %, not unknown.

### CODE REVIEW

**Comment by gst** [comment]
> should give us, (once the coveralls.io workers will process it), a coveralls.io badge with some %, not unknown.

only once another commit will be added after this PR is merged..

File: .travis.yml [hunk 1]
```diff
@@ -20,4 +20,4 @@ script:
   - sh test/test_shebang.sh
 # specific call to launch coverage data into coveralls.io
 after_success:
-  coveralls
+  mv test/.coverage && coveralls --rcfile=test/.coveragerc -v
```
Comment (ddurieux) [inline .travis.yml]: mv test/.coverage ? You don't have forgotten something in the command? :p

Comment (gst) [inline .travis.yml]: .

**Comment by Seb-Solon** [comment]
Commit are all mix now because you push -f on develop yesterday night gst (cf travis build triggered)

Why such a push on devel that lead to that messy PR (importedlib mixed with changelog and coverall)

**Comment by gst** [comment]
> Commit are all mix now because you push -f on develop yesterday night gst (cf travis build triggered)

YOU did it :D cfr : the events notification history in gitter-alignak room , @22:07 (montréal time) yesterday !!

> Why such a push on devel that lead to that messy PR (importedlib mixed with changelog and coverall)

I'm outrageous ;)

File: .travis.yml (hunk 1 above)
Comment (gst) [inline .travis.yml]: gonna correct that but have to sync a bit with the mess of previous evening.. ;)

**Comment by Seb-Solon** [comment]
+1 mea culpa. I just looked at the commiter, no clue in Travis of who pushed that.
I was kinda supried we pushed on alignak at the same time actually :D

**Comment by gst** [comment]
there we go, @ddurieux  it's corrected, I think you can merge ;)

**Pull request merged**